bitcoin-magazine/bitcoin-explained/bitcoin-core-22.0-explained.md

```diff
@@ -0,0 +1,867 @@
+
+Aaron: 00:17:39
+
+Yes.
+So what does Bitcoin Core 22 include taproot wise?
+
+Sjors: 00:17:44
+
+Well, it includes some very rudimentary support for wallet, taproot wallets.
+Basically you're just a single public key type of wallet, which doesn't make taproot that interesting.
+And slightly more generic things where you can have one private one yeah one key per taproot branch so remember in taproot you have a main key and you also have scripts that are sitting in a tree.
+And so there's now support for those scripts in a tree.
+However, as far as I understand it, those scripts can only be of the super simple type, namely a public key.
+
+Aaron: 00:18:27
+
+Right.
+
+Sjors: 00:18:28
+
+So, they're not very interesting yet, but this is how it goes with Bitcoin Core.
+We add little increments at a time.
+
+Aaron: 00:18:35
+
+Yes, we've mentioned that a couple of times.
+Bitcoin Core is improved slowly but surely.
+Every half year there's a new release, so it's not necessarily that all the taproot stuff comes at the same time.
+It's just step by step.
+And now the first step is to have very basic taproot support, which essentially comes down to taproot support for regular transactions.
+Yeah.
+So none of the fancy smart contract stuff, but you can use a Taproot type of address and transaction to receive funds.
+
+Sjors: 00:19:07
+
+Yeah, in principle, but again, it's not even on by default.
+You actually have to manually make it happen.
+So probably for version 23, it'll automatically do the Taproot things.
+
+Aaron: 00:19:19
+
+Yeah.
+I think another way to put it short, if you, if you agree with me, is that there's no actual benefit yet in using Taproot in this Bitcoin Core release, but it's possible to use it for sort of regular transactions, right?
+
+Sjors: 00:19:31
+
+Yeah, exactly.
+So you can if you really know what you're doing.
+And other than that, it's probably useful to have it in there anyway for people who want to test things on SIGNET or TESNET or you know that sort of stuff.
+Because we want other developers to keep working on Taproot.
+
+Aaron: 00:19:44
+
+Yeah, so this was basically Taproot addresses we were discussing.
+A little bit deeper under the hood, you have the taproot descriptors.
+Descriptors?
+What are taproot descriptors?
+
+Sjors: 00:19:55
+
+Yes, so a descriptor is a way to say I want to generate a series of addresses, basically.
+So instead of saying, okay, our wallet is one address, you say, okay, our wallet starts with the root private key, and then it does a bunch of BIP32 style derivations, and then it says, okay, give me the address number 15 or 16 or 17.
+And descriptors allow you to describe that.
+And now they also let you do that for Taproot.
+
+Aaron: 00:20:25
+
+I think descriptors also allow you to sort of categorize your funds into specific types of funds, right?
+So you
+
+Sjors: 00:20:34
+
+Hmm hmm
+
```

Comment:
This kind of lines don't contribute to the transcript's content - you can just delete this and continue with the same speaker. This transcript has frequent back-and-forth between participants, so you'll notice the presence of numerous filler words or non-essential lines that need removal.